src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
```

Comment:
can be removed since https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx is handling this